arm-documentdb/2015-04-08/swagger/documentdb.json

```diff
@@ -312,6 +312,7 @@
       "post": {
         "operationId": "DatabaseAccounts_RegenerateKey",
         "description": "Regenerates an access key for the specified Azure DocumentDB database account.",
+        "x-ms-long-running-operation": true,
```

Comment:
IF regenerateKey is a long running operation then what is the terminal status code? I mean there should be a 200 documented as well, that would indicate AutoRest to deserialized the response based on some schema.